### PR TITLE
Update to latest networks

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
- Removed deprecated networks on the dapp
- Remove duplicated graphics inside create account modal
- Remove throw error inside use currency balance hook
- Display icon and add info icon for notification item

### Reference issue to close (if applicable)
- Closes https://github.com/webb-tools/webb-dapp/issues/704
- Closes https://github.com/webb-tools/webb-dapp/issues/706

-----
### Code Checklist 

- [x] Tested
- [ ] Documented

### CODE REVIEW

File: libs/api-provider-environment/src/WebbProvider.tsx [hunk 1]
```diff
@@ -488,21 +488,25 @@ export const WebbProvider: FC<WebbProviderProps> = ({ children, appEvent }) => {
...
-                    'https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Kovan]:
-                    'https://kovan.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Rinkeby]:
-                    'https://rinkeby.infura.io/v3/e54b7176271840f9ba62e842ff5d6db4',
+
                   //default on metamask
                   [EVMChainId.Beresheet]: 'http://beresheet1.edgewa.re:9933',
+
                   [EVMChainId.HarmonyTestnet1]: 'https://api.s1.b.hmny.io',
```
Comment (dutterbutter) [inline libs/api-provider-environment/src/WebbProvider.tsx:510]: We can remove these as well.

File: libs/dapp-types/src/ChainId.ts [hunk 2]
```diff
@@ -45,30 +42,91 @@ export enum EVMChainId {
...
+
+  HarmonyTestnet1 = calculateTypedChainId(
+    ChainType.EVM,
+    EVMChainId.HarmonyTestnet1
+  ),
+
+  HarmonyTestnet0 = calculateTypedChainId(
+    ChainType.EVM,
+    EVMChainId.HarmonyTestnet0
+  ),
```
Comment (dutterbutter) [inline libs/dapp-types/src/ChainId.ts:62]: I think we only need one harmony testnet, no? What are the differences?

Comment (AtelyPham) [reply libs/dapp-types/src/ChainId.ts:62]: Can you help me to confirm this, please? This config is existing before I edited this file. @AhmedKorim

Comment (AhmedKorim) [reply libs/dapp-types/src/ChainId.ts:61]: we should  keep both, 0,1 are related to network shard https://docs.harmony.one/home/general/ecosystem/wallets/browser-extensions-wallets/others/adding-harmony

File: libs/api-provider-environment/src/WebbProvider.tsx [hunk 3]
```diff
@@ -488,22 +488,21 @@ export const WebbProvider: FC<WebbProviderProps> = ({ children, appEvent }) => {
...
-                    'https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Goerli]:
-                    'https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Kovan]:
-                    'https://kovan.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Rinkeby]:
-                    'https://rinkeby.infura.io/v3/e54b7176271840f9ba62e842ff5d6db4',
-                  //default on metamask
-                  [EVMChainId.Beresheet]: 'http://beresheet1.edgewa.re:9933',
-                  [EVMChainId.HarmonyTestnet1]: 'https://api.s1.b.hmny.io',
```
Comment (AhmedKorim) [inline libs/api-provider-environment/src/WebbProvider.tsx:506]: we can keep `Harmony1` and `Goerli`

File: libs/api-provider-environment/src/error/interactive-errors/unsupported-chain.ts [hunk 4]
```diff
@@ -15,7 +15,6 @@ export function unsupportedChain(apiConfig: ApiConfig): InteractiveFeedback {
     },
     {
       list: [
-        apiConfig.getEVMChainName(EVMChainId.Rinkeby),
         apiConfig.getEVMChainName(EVMChainId.Beresheet),
```
Comment (AhmedKorim) [inline libs/api-provider-environment/src/error/interactive-errors/unsupported-chain.ts:18]: We can remove Beresheet @dutterbutter  ?

**Review by AhmedKorim — CHANGES_REQUESTED** [review]

File: libs/api-provider-environment/src/WebbProvider.tsx (hunk 3 above)
Comment (dutterbutter) [reply libs/api-provider-environment/src/WebbProvider.tsx:506]: You have to make sure this endpoint is not deprecated first.

**Comment by AtelyPham** [comment]
I have tested all RPCs by using `curl` to query the latest block and compare it with the live block explorer ✅.

**Pull request merged**